examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -755,7 +771,20 @@ impl FrontendNode {
             Err(OakStatus::ErrChannelClosed),
             oak::channel_write(out_channel, &data, &[])
         );
+        expect_eq!(Ok(()), oak::channel_close(out_channel.handle));
 
+        // The same is true even if there are messages pending on the channel.
```

Comment:
This seems a separate test case from the previous one, worth splitting? up to you

Reply:
Yep, split.